setup.py

```diff
@@ -62,7 +62,8 @@
 
 # Only import these if the above checks are okay
 # to avoid masking the real problem with import error.
-from setuptools import setup  # noqa
-from extension_helpers import get_extensions  # noqa
+from setuptools import setup  # noqa: F402
```

Comment:
Typo?

```suggestion
from setuptools import setup  # noqa: E402
```

Reply:
E402 is for ``module level import not at top of file``

Reply:
``Flake8`` imports all the ``pycodestyle`` errors.
https://flake8.pycqa.org/en/latest/user/error-codes.html

Reply:
Yeah, but why is one line marked for F402 and then the next one after that E402?

Reply:
Oh. Yes. My bad. That is indeed a mistake.

Reply:
I saw my one F402 and explained why it's wrong 😆